code/modules/antagonists/wizard/wizard.dm

```diff
@@ -5,6 +5,8 @@
 	success_medal = "You're no Elminster!"
 	faction = list(FACTION_WIZARD)
 	uses_pref_name = FALSE
+	var/list/datum/SWFuplinkspell/purchased_spells = list()
+	var/spell_animation_frame = 5 // This will need updating if the spell animations ever change
```

Comment:
This is fine by the way, wizard spell sprites include the animation baked into them so there isn't really a better way to do this. Could be a local `#define` though since it's just taking up memory here as a var.

Reply:
cries, ok thank you for the help!!